Tp 02, reduce number of clickables

Chandrika-Rani-Infrrd opened the pull request:
updated:table component with two buttons
 updated:table component with modal shared between form and edit

Chandrika-Rani-Infrrd committed:
[TP-02],reduce number of clicks
updated:tablecomponent with two buttons
updated:table component with modal shared between form and edit

Chandrika-Rani-Infrrd committed:
[TP-01],refactor components
added:formFiels component which reduces the htmlcode to a component
added:formButton component that is reused in form and edit
updated:converted HTML code into component of edit
updated:converted HTML code into component of form

azeez-infrrd commented:
Please go through: https://airbnb.io/javascript/react/
and update the code as per the guidelines

shaik-azeezahmed reviewed (changes requested):
let's start following the guidelines for code sanity and re-visit the review.

Chandrika-Rani-Infrrd committed:
[TP-02],code indentation
updated:converted stateless class to function
updated:spacing between the lines

Chandrika-Rani-Infrrd committed:
TP-02,reduce number of clickables
